client/components/ProductOverview/index.js

```diff
@@ -1,18 +1,35 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import { ProductContext } from '../../context/ProductContext';
 import { ImageGallery } from './layouts/ImageGallery';
+import { NewImg } from './layouts/NewImg';
 import { ProductMainInfos } from './layouts/ProductMainInfos';
-import { ProductSubInfo } from './layouts/ProductSubInfo';
 
 export const ProductOverview = () => {
+  const { curProduct } = useContext(ProductContext);
+
   return (
-      <div className="mBody">
-        <div className="mRowA">
+    <div className="d-block">
+      <div className="row">
```

Comment:
Might want to rename to something like mRow or m-row. We will probably have quite a few classes that all want the name row.

Reply:
I already remove the mRow. I have deleted many of my CSS, i'm just going to use bootstrap all the way.